.github/workflows/mobile-build.yml

```diff
@@ -0,0 +1,101 @@
+# Builds unity project and creates iOS and Android executables for sideloading.
+name: Mobile Build
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - '*'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      clean:
+        description: 'Clean the project before building'
+        required: false
+        default: false
+        type: 'boolean'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ ( github.event_name == 'pull_request' || github.event.action == 'synchronize' ) }}
+jobs:
+  build:
+    env:
+      UNITY_PROJECT_PATH: '' # set by unity-setup action
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, macos-13]
```

Comment:
these templates should probably use buildalon by default.

Reply:
No these are specifically for github hosted runners